docs/source/tutorial/tutorial-obtain-schema.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Step 2: Obtain your GraphQL schema"
+---
+
+## Explore your server's schema
+
+This tutorial uses the same GraphQL server that's provided as part of [the Apollo full-stack tutorial](https://www.apollographql.com/docs/tutorial/introduction/). You can visit [`https://n1kqy.sse.codesandbox.io/`](https://n1kqy.sse.codesandbox.io/) to start it up and open the GraphiQL tool to explore its schema: 
```

Comment:
This is...not correct - it's a modified version of the server that you wind up _creating_ in the full-stack tutorial, it is not provided during that tutorial at all. 

For issues like this, would you prefer I address this after merge and then have you take another quick run-through?

Reply:
Issues like this are great to resolve before merge! Resolving now. And yes I am bound to make more edits along the way that are not correct; please do not hesitate to let me know when I do :D